pkg/controller/account/byoc.go

```diff
@@ -408,8 +472,14 @@ func (r *ReconcileAccount) getCCSClient(currentAcct *awsv1alpha1.Account, accoun
 	return ccsAWSClient, nil
 }
 
-func validateBYOCClaim(accountClaim *awsv1alpha1.AccountClaim) error {
+func validateSTSClaim(accountClaim *awsv1alpha1.AccountClaim) error {
+	if accountClaim.Spec.BYOCAWSAccountID == "" {
```

Comment:
Should we also validate if `STSRoleARN` is empty?

Reply:
Good catch.  Updated to actually validate what matters for an STS account claim.